bundles/org.openhab.binding.bluetooth/README.md

```diff
@@ -19,10 +19,11 @@ This should be the best choice for any Linux-based single board computers like e
 
 Two thing types are supported by this binding:
 
-| Thing Type ID | Description                                                                                             |
-|---------------|---------------------------------------------------------------------------------------------------------|
-| beacon        | A Bluetooth device that is not connected, but only broadcasts annoucements.                             |
-| connected     | A Bluetooth device that allows a direct connection and which provides specific services when connected. |
+| Thing Type ID | Description                                                                                                |
+|---------------|------------------------------------------------------------------------------------------------------------|
+| beacon        | A Bluetooth device that is not connected, but only broadcasts annoucements.                                |
+| connected     | A Bluetooth device that allows a direct connection and which provides specific services when connected.    |
+| roaming       | A virtual Bluetooth adapter that interacts with Bluetooth devices through their nearest Bluetooth adapter. |
```

Comment:
This is not the right place to add it - if I get it right, "roaming" is a new adapter (next to BlueZ and BlueGiga) and not a BT device type (beacon or connected).

Reply:
You are correct, "roaming" is a type of adapter, not a regular bluetooth device. I just put it in this table since I thought all thing types should be listed there. Should I create a separate section/table for the roaming adapter?

Reply:
As mentioned further below, this should ideally be a separate bundle, just like the other adapters.